### PR TITLE
User_request_new_book_and_show_list_request

### PR DESCRIPTION
**Pull request by samnangcattor**

### CODE REVIEW

File: app/controllers/request_books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    else
+      render "new"
+    end
+  end
+
+  def edit
+  end
+
+  def update 
+    if @request_book.update_attributes request_params
```
Comment (HorNorin) [inline app/controllers/request_books_controller.rb]: no else clause?

Comment (samnangcattor) [inline app/controllers/request_books_controller.rb]: Look at   the line 30 and 31

**Comment by HorNorin** [comment]
ok

**Comment by RathanakSreang** [comment]
ok

**Pull request merged**